corgidrp/mocks.py

```diff
@@ -71,10 +72,52 @@ def create_nonlinear_dataset(filedir=None, numfiles=2,em_gain=2000):
 
 
 
-def create_default_headers():
+def create_prescan_files(filedir=None, numfiles=2, obstype="SCI"):
+    """
+    Create simulated raw data. 
+
+    Args:
+        filedir (str): (Optional) Full path to directory to save to.
+        numfiles (int): Number of files in dataset.  Defaults to 2.
+        obstype (str): Observation type. Defaults to "ENG".
+
+    Returns:
+        corgidrp.data.Dataset:
+            The simulated dataset
+    """
+    
+    if obstype == "SCI":
+        size = (1200, 2200)
+    elif obstype == "ENG":
+        size = (2200, 2200)
+    elif obstype == "CAL":
+        size = (2200,2200)
+    else:
+        raise ValueError(f'Obstype {obstype} not in ["SCI","ENG","CAL"]')
```

Comment:
"CAL" should be taken out.

Reply:
Oh, maybe the "CAL" case will be added later?

Reply:
I think we need to coordinate with @agreenbaum on exactly what all the different OBSTYPE categories will be for the real data.

Reply:
But maybe not something to figure out right now for this PR